dashboard/app/main.go

```diff
@@ -282,6 +295,26 @@ func handleInvalid(c context.Context, w http.ResponseWriter, r *http.Request) er
 	})
 }
 
+func handleRepos(c context.Context, w http.ResponseWriter, r *http.Request) error {
+	hdr, err := commonHeader(c, r, w, "")
+	if err != nil {
+		return err
+	}
+	repos := []*uiRepo{}
+	for _, repo := range config.Namespaces[hdr.Namespace].Repos {
```

Comment:
Do we poll these repos in this context?
The main source of commits if the manager builds. I don't remember if presence of the commit in any of config.Namespaces[hdr.Namespace].Repos would prevent the message. But if it does, then we still need to merge manager repos with the config list.

Reply:
It seems to me that we take them from `Repos`.

Here we request the list of repositories to poll: https://github.com/google/syzkaller/blob/master/syz-ci/jobs.go#L115
Here we form that list from the `Repos` field: https://github.com/google/syzkaller/blob/master/dashboard/app/api.go#L231

I cannot find where we additionally poll the manager repo.

Reply:
We detect commits that present in builds here:
https://github.com/google/syzkaller/blob/master/syz-ci/manager.go#L638

Reply:
Ok, so we need to take `Repos` and add repositories from the latest manager builds there.

Reply:
If the commit is present only in config.Namespaces[hdr.Namespace].Repos, but not in any of the manager builds, does it prevent the syzbot notification?

Reply:
Hmmm, yes, you're right. We use `Repros` to poll fixing commits, but we only save them to `PatchedOn` if they're reported with a successful manager build. 

So, we only need the repositories from the managers (from the dashboard side, I can only take them from the latest manager builds).

Reply:
Done